### ISSUE TITLE
Player Simulator voiding some items

### ISSUE THREAD

**Issue by KAfable** [CyclopsMC/IntegratedTunnels#68]
Hi,

I'm having a little bit of trouble trying to get the Player Simulator to work with Botania's Horn of the Wilds. Here's my set up

I have a chest with Botania's Horn of the Wild (when held down in a right click by a player, it'll break all grasses and harvest crops, dropping them items onto the ground, I know the Drum of the Wild exists, but it isn't very fast/powerful compared to aa automated Horn of the Wild).

Attached to the chest is an inventory interface with logic cables leading to a Player Simulator. Inside the player simulator, I placed a Variable Card set to "Item: Horn of the Wild" via Logic Programmer, and placed it in the "Click Item" section of the Player Simulator, where it gave me a green checkmark. I edited the Click Item configs to be true to 'right click' and true to 'continuous click' and left everything else.

The expected result is that the Horn would say in the chest, and be used by the Player Simulator to break all of the nearby grass, the reality was that the horn was used up / voided by the player simulator and nothing happened. Any idea what I did wrong?

**Comment by Vyraal1** [comment]
In another test I was using Cyclic's Evoker Fang Wand. This time I inserted a variable card programmed to an Evoker Fang with 100% durability. The wand mimics the Evoker mob's ability by shooting out entities that damage mobs when you right click the ground with it, I inserted the programmed variable card into the "Click Item" slot, and then set right click to true, and continuous click to false.

It used the wand once (but consumed it as well, whereas one wand usually has 1k uses or so), however it wouldn't use any more other wands left in the chest. I had to rebuild the setup from scratch to get it use one wand before breaking.
